### PR TITLE
MAISTRA-2089: Update templates for 2.0.2

### PR DESCRIPTION
**Pull request by knrc**

### CODE REVIEW

File: resources/smcp-templates/v1.1/base [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: ServiceMeshControlPlane
 spec:
   istio:
     global:
-      tag: 1.1.11
+      tag: 1.1.12
```
Comment (jwendell) [inline resources/smcp-templates/v1.1/base:6]: Do you also need to update v2.0 files?

Comment (knrc) [reply resources/smcp-templates/v1.1/base:6]: Yes, good catch

Comment (knrc) [reply resources/smcp-templates/v1.1/base:6]: I've just checked and it doesn't seem to be included in the 2.0 base

Comment (knrc) [reply resources/smcp-templates/v1.1/base:6]: sorry, it is but a different name.  I'll update now

**Review by brian-avery — APPROVED** [review]

File: resources/smcp-templates/v1.1/base (hunk 1 above)
Comment (knrc) [reply resources/smcp-templates/v1.1/base:6]: @jwendell can you check again?  I've just pushed up the 2.0 changes matching the above

**Comment by knrc** [comment]
There appears to be a number of test failures, on the surface they do not appear to be related but I'm investigating

**Review by jwendell — APPROVED** [review]

**Comment by knrc** [comment]
I've pushed an update for one test, most of the others appear to be warnings about a retry.

**Comment by jwendell** [comment]
> I've pushed an update for one test, most of the others appear to be warnings about a retry.

It worked \o/

Next time just push another commit, it makes our lives easier to review.

**Pull request merged**